data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,426 @@ mission "Remnant: Face to Maw 2B"
+			label clarification
+			choice
+				`	"Wait, you didn't sell any of that?"`
+				`	"Okay, so that shipment you had me get?"`
+					goto crystaljobs
+			label crystaljobs
+			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to gauge your willingness to help. Your shipment was definitely useful, but we had to ensure we framed your job in a way that made sense without revealing what we were actually doing.`
+			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
+			choice
+				`	(Sign in the affirmative.)`
+				`	(Decline.)`
+					decline
+			`	"Great. They would like you to pick up another 19 Keystones. Two differences this time: First, we would like you to pick them up from several different planets. And second, by the time you reach Hai space, we will have made arrangements so the shipments will be already paid for and waiting for you. Pick up the first stones on Allhome, then the next set on Greenwater. I will have a list sent to you by the time you reach Greenwater."`
+			`	"That being said, if you happen to pick up any additional crystals, you can turn them into the logistics office. They will handle the distribution and compensate you accordingly."`
+			choice
+				`	"Wait, how will you get this all set up?"`
+				`	"Sounds good."`
+					accept
+			branch remnantuntrusted
+				"remnantuntrusted" >= 1 # This attribute is currently not used, but will be set by some of the "consequence" missions as a result of spying for the Deep.
+			`	Aeolus looks faintly amused. "While you are certainly adventurous, you are not the only one to have traveled widely. We mapped the Hai systems long ago with cloaked ships, but never landed or contacted any of them. So we never knew anything more than what we could observe from orbit or from scanning ships. But you are known to them, and have a fairly public reputation. By using your transponder codes, we can place and pay for orders through hyperspace relays. All we need is a ship in position to feed the signal into the network.`
+				accept
+			label remnantuntrusted
+			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
+				accept
+	on complete
+		dialog
+			`It takes a few minutes to contact the yardmaster, but once you do, a mixed crew of Hai and Human dockworkers have the crates moved promptly from the warehouse to your cargo bay. Hai and Humans seem accustomed to working together here.`
+
+
+
+mission "Remnant: Crystal Research 2"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
```

Comment:
Because the player *may* be arriving via JD instead of through the wormhole; and it is likely the Remnant set it up from a more holistic view as a sweep of stops across the area instead of going by what is easiest for an HD.